client/src/pages/ChefContent.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import { Edit } from "@material-ui/icons";
+import Paper from "@material-ui/core/Paper";
+import Avatar from "@material-ui/core/Avatar";
+import Divider from "@material-ui/core/Divider";
```

Comment:
All of the core components can be imported together
```suggestion
import { Button, Typography, Grid, Paper, Avatar, Divider } from "@material-ui/core";
```

Reply:
i'll import them together